native/core/src/execution/datafusion/planner.rs

```diff
@@ -1234,24 +1233,51 @@ impl PhysicalPlanner {
     ) -> Result<Arc<dyn AggregateExpr>, ExecutionError> {
         match spark_expr.expr_struct.as_ref().unwrap() {
             AggExprStruct::Count(expr) => {
-                let children = expr
-                    .children
-                    .iter()
-                    .map(|child| self.create_expr(child, schema.clone()))
-                    .collect::<Result<Vec<_>, _>>()?;
+                if expr.children.iter().len() == 1 {
```

Comment:
Hmm, I think we can also do this for multiple child expressions?

Reply:
Thanks. I have extended this approach for the multiple argument case.